alma/alma-woocommerce-gateway
# Display deferred payments with in page

## Reason for change



[Task](https://linear.app/almapay/issue/ECOM-1461/[%F0%9F%A7%A9-wc]-add-gateway-to-display-deferred-payments-with-in-page)

### How to test

- Deploy woocommerce 8-2-2
- activate in page and deferred payment
- check the checkout page and make a payment
